x11-plugins/gkrellm-cpupower/files/gkrellm-cpupower-0.2-makefile.patch

```diff
@@ -0,0 +1,35 @@
+--- a/Makefile
++++ b/Makefile
+@@ -3,17 +3,22 @@
+ GTK_INCLUDE = `pkg-config gtk+-2.0 --cflags`
+ GTK_LIB = `pkg-config gtk+-2.0 --libs`
+ 
+-FLAGS = -O2 -Wall -fPIC $(GTK_INCLUDE)
++FLAGS = -fPIC $(GTK_INCLUDE)
+ LIBS = $(GTK_LIB)
+ 
+ LFLAGS = -shared -lcpupower
+ 
+-CC = gcc $(CFLAGS) $(FLAGS)
++CC = $(CC)
+ 
+ OBJS = cpupower.o
+ 
++all: cpupower.so
++
++%.o: %.c
++	$(CC) $(CFLAGS) $(FLAGS) -c -o $@ $<
++
+ cpupower.so: $(OBJS)
+-	$(CC) $(OBJS) -o cpupower.so $(LFLAGS) $(LIBS)
++	$(CC) $(CFLAGS) $(LDFLAGS) $(FLAGS) $(OBJS) -o cpupower.so $(LFLAGS) $(LIBS)
+ 
+ install: cpupower.so
+ 	install -D -m 755 -s cpupower.so $(DESTDIR)/usr/lib/gkrellm2/plugins/cpupower.so
```

Comment:
It seems odd to me that it's always `lib` but I might be missing something. Also, doubt `-s` should be here (strip).

Reply:
I'm not sure about the destination here, but I've gone ahead and removed `-s`.

Reply:
I'll remember to check that for the other Makefiles.